gammapy/astro/darkmatter/utils.py

```diff
@@ -26,19 +27,36 @@ def __init__(self, geom, profile, distance):
         self.profile = profile
         self.distance = distance
 
-    def compute_differential_jfactor(self):
+    def compute_differential_jfactor(self, ndecade=1e4):
         r"""Compute differential J-Factor.
 
         .. math::
             \frac{\mathrm d J}{\mathrm d \Omega} =
             \int_{\mathrm{LoS}} \mathrm d r \rho(r)
```

Comment:
This should read \rho(r)**2. 
I know, it's nothing to do with the changes done here, but when we first review it, we can just correct this as well, I think.

Reply:
I made the correction